sass/oscailte/homepage/_home.scss

```diff
@@ -1,3 +1,13 @@
+ul.menu li {
```

Comment:
Please remove all menu changes from this PR and limit the scope of this PR to fixing the border below the iPhone image. 

We will use #19052 to fix the menu.

Reply:
The border below the iPhone is already fixed in the current with a CSS change, I think by @frenck . So this PR can be closed.